src/test/java/seedu/address/logic/parser/meeting/DeleteMeetingCommandParserTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser.meeting;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_MEETING;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.meeting.DeleteMeetingCommand;
+
+/**
+ * As we are only doing white-box testing, our test cases do not cover path variations
+ * outside of the DeleteCommand code. For example, inputs "1" and "1 abc" take the
+ * same path through the DeleteCommand, and therefore we test only one of them.
```

Comment:
DeleteCommand should be DeleteMeetingCommand right?

Reply:
fixed